extensions/mssql/src/contracts.ts

```diff
@@ -1024,3 +1024,21 @@ export namespace GetSqlMigrationAssessmentItemsRequest {
 }
 
 // ------------------------------- <Sql Migration> -----------------------------
+
+// ------------------------------- < Table Designer > ------------------------------------
+
+export interface ProcessTableDesignerEditRequestParams {
```

Comment:
I don't think we've typically added the "Process" prefix on these types of classes.  Maybe more consistent to be something like `TableDesignerEditRequestParams`.